dev/com.ibm.ws.microprofile.config.1.3_fat/publish/servers/MapEnvVarServer/bootstrap.properties

```diff
@@ -0,0 +1,3 @@
+bootstrap.include=../testports.properties
+osgi.console=7773
+com.ibm.ws.logging.trace.specification=APPCONFIG=all
```

Comment:
I realise that this was copied from the other test but I think the new strategy was to leave trace turned off unless we really need it. Perhaps we should turn it off in both new tests?